drivers/fpga/fpga_altera.h

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2024, Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SUBSYS_FPGA_BRIDGE_INTEL_H_
+#define ZEPHYR_SUBSYS_FPGA_BRIDGE_INTEL_H_
+
+#include <zephyr/kernel.h>
+#include <zephyr/sip_svc/sip_svc.h>
+#include <zephyr/drivers/sip_svc/sip_svc_agilex_smc.h>
```

Comment:
Are these two headers required by anything in this file? If not, please include them where they are needed.

Reply:
done